### ISSUE TITLE
Query Overview page not loading when trying to view details of query with session params

### ISSUE THREAD

**Issue by OleksiiDuzhyi** [prestodb/presto#7634]
When you try to access Query Overview page from Cluster Overview for query with session params, an error occurs: 

```
Uncaught ReferenceError: property is not defined
    at t.renderSessionProperties (eval at transform.run (browser.min.js:1), <anonymous>:1253:22)
    at t.render (eval at transform.run (browser.min.js:1), <anonymous>:1636:42)
    at c._renderValidatedComponentWithoutOwnerOrContext (react-15.0.1.min.js:13)
    at c._renderValidatedComponent (react-15.0.1.min.js:13)
    at c._updateRenderedComponent (react-15.0.1.min.js:13)
    at c._performComponentUpdate (react-15.0.1.min.js:13)
    at c.updateComponent (react-15.0.1.min.js:13)
    at c.performUpdateIfNecessary (react-15.0.1.min.js:13)
    at Object.performUpdateIfNecessary (react-15.0.1.min.js:15)
    at u (react-15.0.1.min.js:15)
```

This started to occur after after changes in `renderSessionProperties` function from [query.js](https://github.com/prestodb/presto/blob/master/presto-main/src/main/resources/webapp/assets/query.js#L846). 

In the first for loop it used to be written like this: 
 `for (var property in query.session.systemProperties) {`

So property variable was visible inside function scope. Now `var` is changed to `let` so property variable is visible only inside this for loop. So when it gets to [query.js:856](https://github.com/prestodb/presto/blob/master/presto-main/src/main/resources/webapp/assets/query.js#L856) Uncaught ReferenceError is thrown.

**Comment by OleksiiDuzhyi** [comment]
#7633

**Comment by martint** [comment]
Fixed in upcoming 0.170 release.
